locator/token_metadata.cc

```diff
@@ -1399,6 +1423,31 @@ future<> token_metadata_impl::calculate_pending_ranges_for_leaving(
     });
 }
 
+future<> token_metadata_impl::calculate_pending_ranges_for_replacing(
+        token_metadata& unpimplified_this,
```

Comment:
`unpimplified_this` is a horrible name. I don't even know what it is supposed to mean

Reply:
It is invented by avi when impl token_metadata.